setup.py

```diff
@@ -32,7 +32,7 @@ def read_file(filename, encoding='utf8'):
 
 install_requires = [
     'acme=={0}'.format(version),
-    'ConfigArgParse',
+    'ConfigArgParse>=0.10.0',  # python2.6 support, upstream #17
```

Comment:
Hmm.  Dynamically-computed `install_requires` causes problems for users of pip >= 7.0, which has wheel caching enabled by default: https://github.com/pypa/pip/issues/3025.  Short version:
- I pip install letsencrypt on Python 2.7, pip runs setup.py and builds a wheel (marked -py2-none-any.whl) and puts it in ~/.cache/pip/
- I pip install letsencrypt on Python 2.6, pip discovers a valid wheel and installs it, using `install_requires` computed earlier for a different Python version.

What's the harm of having everyone use a recent version of ConfigArgParse?